x/avs/keeper/keeper.go

```diff
@@ -382,6 +384,17 @@ func (k Keeper) IterateAVSInfo(ctx sdk.Context, fn func(index int64, avsInfo typ
 	}
 }
 
+func (k Keeper) GetAVSEpochInfo(ctx sdk.Context, addr string) (*epochstypes.EpochInfo, error) {
+	avsInfoResp, err := k.GetAVSInfo(ctx, addr)
+	if err != nil {
+		return nil, err
+	}
+	avsInfo := avsInfoResp.Info
+	// Epoch information must be present because it is checked when setting AVS information.
+	epochInfo, _ := k.epochsKeeper.GetEpochInfo(ctx, avsInfo.EpochIdentifier)
+	return &epochInfo, nil
+}
```

Comment:
It’s unnecessary to check the AVS address because this function is called internally and will return an error if an invalid address is input. Regarding the `found` check, it is already guarded when setting, so we don't need to check it here.